flipperdevices/flipperzero-firmware
# [FL-2556] Update complete screen

# What's new

- [FL-2556] Added a post-update screen 
- Added support for slideshows that are rendered by Desktop application on device startup
- Slideshows contain one or more compressed images in a single file `/int/slideshow` 
- Slideshows can be navigated with < and > (+"ok") buttons in both directions
- Slideshow can be dismissed with "back" button
- Slideshows are deleted after being completed or dismissed
- Slideshows are baked with `scripts/slideshow.py` from a directory of "frame_XX.png" images 

# Verification 

- Install update package with web updater or manually
- Check that after all operations are completed, a special image is displayed 
- Check that the image is only displayed once
- (Optionally) Build a custom slideshow binary from multiple images, upload it, restart device and check navigation in both directions

# Checklist (For Reviewer)



[FL-2556]: https://flipperzero.atlassian.net/browse/FL-2556?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
